.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -41,7 +41,7 @@ The ARM reviewer on-call engineer visits the merge queue twice a day, so the app
 What's the purpose of this PR? Check the specific option that applies. This is **mandatory**!
 
   - [ ] New resource provider.
-  - [ ] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been generated using [OpenAPI Hub](https://aka.ms/openapihub)).
+  - [ ] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been generated using [OpenAPI specs automation](https://aka.ms/azsdkdocs/createopenapispec)).
```

Comment:
> (If API spec is not defined in TypeSpec, the PR should have been generated using [OpenAPI specs automation]

I think this is incorrect: there is now no OpenAPI specs automation generating the PR. Instead, the PR needs to be created manually, by following the OpenAPI specs PR creation guidance.

Reply:
well, there is a script that automates the creation of the first 3 commits, which are required by arm team. I do like your proposal better so it is more generic